definitions/variable/tag_fe_carriers.yaml

```diff
@@ -0,0 +1,95 @@
+# This list of inputs will replace any occurrence of `{Final Energy carrier}`
+# in a variable-definition yaml file
+
+- Final Energy carrier:
+  - Electricity:
+      description: electricity (including on-site solar PV), excluding transmission/distribution losses
+      weight: Electricity
```

Comment:
The intention was to allow the weight definition for price variables: https://github.com/openENTRANCE/openentrance/pull/166#issuecomment-1080769555

For example:
```
- Price|Secondary Energy|{Final Energy carrier}:
    description: price at the secondary level of {Final Energy carrier}, i.e. for large scale
                consumers (e.g. aluminum production, gas power plant)
    notes: Prices should include the effect of carbon prices.
    unit: [EUR_2020/GJ, USD_2010/GJ]
    weight: Secondary Energy|{Final Energy carrier}
```
All quantity variables contained in this pull request does not require any weight by definition. This only added to support the weight overwrite on cost and price cariables from other pull requests.

Reply:
Ah ok, then all good, never mind.

Reply:
Regarding the unintended additional variables, I definitely agree with you about the exclusion list being a more viable option. 

I would say that we have a hybrid of nonsensical variables and variables excluded due to tractability reasons.

Below some examples of nonsensical variables:

- ex 1. all electricity use is energy related, so it will never make sense to have the variable `Final Energy|Non-Energy Use|Electricity` as far as I know. 
- ex 2. The {Final Energy carrier} tag was not used to create the emission variables in the emissions pull request (https://github.com/openENTRANCE/openentrance/pull/163) because it would create a lot of nonsensical emission variables, such as `Emissions|CO2|Energy|Demand|Transportation|Electricity`, which is clearly wrong as BEVs don't emit CO2 when using the electricity in the bateries. 

However, there are some variables that are semantically perfect fine, but are not included in the current variables templates like NAVIGATE to avoid unnecessary duplications (see example 3 below), or just for practicality of the cariables template size (example 4):

- ex 3. `Transportation|2W and 3W|Passenger` is just a duplication of `Transportation|2W and 3W`, so to avoid exploding the number of variables reported, models and variables templates only include `Transportation|2W and 3W` usually.
- ex 4. `Emissions|CO2|Energy|Demand|Transportation|{Transport mode, tech and carrier}`

Reply:
Ok right, thanks for the quick clarification, then this definitely sounds like an issue/feature request for the nomenclature package.
The main question would be on which side we should implement this, the tag side or the variable side, but we can move this discussion over to a dedicated issue.